beetsplug/export.py

```diff
@@ -117,35 +145,79 @@ def run(self, lib, opts, args):
 
 class ExportFormat(object):
     """The output format type"""
+    def __init__(self, file_path, file_mode=u'w', encoding=u'utf-8'):
+        self.path = file_path
+        self.mode = file_mode
+        self.encoding = encoding
+        """ self.out_stream =
+                sys.stdout if path doesn't exit
+                codecs.open(..) else
+        """
```

Comment:
Perhaps this is out of date? Or it can be rephrased as a natural-language comment?

Reply:
Here is the new comment...
`# creates a file object to write/append or sets to stdout`